scripts/base/protocols/redis/main.zeek

```diff
@@ -139,10 +173,29 @@ event Redis::command(c: connection, cmd: Command)
 		}
 
 	++c$redis_state$current_command;
+
+	if ( cmd?$known )
+		{
+		if ( c$redis_state$resp_version == RESP2
+		    && cmd$known in enter_subscribed_mode )
+			{
+			c$redis_state$subscribed_mode = T;
+			}
+		if ( c$redis_state$resp_version == RESP2
+		    && cmd$known in exit_subscribed_mode )
+			{
+			c$redis_state$subscribed_mode = F;
```

Comment:
Should this report a warning or something if the command is in both sets? Also, the `resp_version` check could just wrap the two `known` checks in a single `if`.

Reply:
> Should this report a warning or something if the command is in both sets?

Sure, that works. It's currently not marked `&redef` but it would be worth a warning if it happened